### DISCUSSION TITLE
Welcome to pinia Discussions!

### DISCUSSION THREAD

**Discussion by posva** [vuejs/pinia#380] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alirezahamid** [comment]
Hello there,
is there any opportunity that i can translate the Pinia documentation to Farsi(Persian)? like adding a new language to the official website?

**Comment by isimmons** [comment]
Hello, my name is Ian. I am learning Pinia. I've also been learning Vue, React, and Typescript for a while because apparently I enjoy pulling my hair out.  Hopefully one day I will know enough to contribute. Thank you for keeping 'discussions' as part of your repo.
